src/app/pages/add-contact/addContact.component.ts

```diff
@@ -31,30 +31,49 @@ export class AddContactComponent implements OnInit {
     }
 
   isEmailInValid: boolean = true;
-  invalidAddContact: boolean = false;
+  isMobileNumberInValid: boolean = true;
+  isHomeNumberInValid: boolean = true;
+  isWorkNumberInValid: boolean = true;
+  invalidAddContact: boolean = true;
```

Comment:
I don't ever see this as false. Your catch at the bottom of the file sets this to true even though by default it is true.